src/rustc/fortanix-sgx-abi_shim/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "fortanix-sgx-abi"
+version = "0.0.0"
+authors = ["The Rust Project Developers"]
+
+[lib]
+path = "rust-sgx/fortanix-sgx-abi/src/lib.rs"
```

Comment:
This crate is just a [single file](https://github.com/fortanix/rust-sgx/blob/master/fortanix-sgx-abi/src/lib.rs) that is a specification of the usercalls. It's not possible to include this from crates.io, is it? Alternatively the file can just be copied in here, or another git submodule can be added. I'm not sure what the best solution is here.

Reply:
Oh sorry, stick to what's in this PR for now. If this lands before #56092 that's fine. If #56092 lands first then we can update this with that strategy.